doc/testing.md

```diff
@@ -378,6 +378,11 @@ Defaults to 4.
 Sets the argument `-timeoutHandlerTimeout` for JTReg. The default value is 0.
 This is only valid if the failure handler is built.
 
+#### JTREG_TEST_THREAD_FACTORY
+
+Sets the `-testThreadFactory` for JTReg. The source code of all plugins is located
+in `test/jtreg_test_thread_factory/'. The plugins are built as a part of test image.
```

Comment:
I think this should probably even include a brief note about what possible values, in context of the JDK build, are applicable. For example the value `Virtual` and what that value means.